### ISSUE TITLE
Cura 3.6.0 Crashes Upon Startup

### ISSUE THREAD

**Issue by efyoung** [Ultimaker/Cura#5074]
**Application Version**
Ultimaker Cura 3.6.0

**Platform**
Windows 8.1, 64-bit Operating System, x64 based processor
Installed memory (RAM): 16.0 GB
Processor: Intel(R) Core(TM) i7-4700MQ CPU @ 2.40 GHz
Graphics Card: Intel(R) HD Graphics 4600 (all drivers up-to-date)

**Printer**
Monoprice Maker Ultimate 3D Printer - MK11 Direct Driver Extruder / 24V Power System

**Steps to Reproduce**
Start Cura

**Actual Results**
Splash screen appears, User agreement appears. Then Cura crashes - Cura.exe has stopped working

**Expected results**
Cura starts

**Additional Information**
 **cura.log -->**
2018-12-27 17:56:14,395 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin FileLogger
2018-12-27 17:56:14,448 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin FirmwareUpdateChecker
2018-12-27 17:56:14,486 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin FirmwareUpdater
2018-12-27 17:56:14,522 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeGzReader
2018-12-27 17:56:14,557 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeGzWriter
2018-12-27 17:56:14,593 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeProfileReader
2018-12-27 17:56:14,675 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeReader
2018-12-27 17:56:14,716 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeWriter
2018-12-27 17:56:14,765 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin ImageReader
2018-12-27 17:56:14,803 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin LegacyProfileReader
2018-12-27 17:56:14,840 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin LocalContainerProvider
2018-12-27 17:56:14,885 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin LocalFileOutputDevice
2018-12-27 17:56:14,920 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin MachineSettingsAction
2018-12-27 17:56:14,956 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin ModelChecker
2018-12-27 17:56:14,995 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin MonitorStage
2018-12-27 17:56:15,037 - WARNING - [MainThread] UM.Preferences.getValue [97]: Tried to get the value of non-existing setting cura/active_mode.
2018-12-27 17:56:15,047 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin PerObjectSettingsTool
2018-12-27 17:56:15,091 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin PostProcessingPlugin
2018-12-27 17:56:15,128 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin PrepareStage
2018-12-27 17:56:15,193 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin RemovableDriveOutputDevice
2018-12-27 17:56:15,278 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SimulationView
2018-12-27 17:56:15,327 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SliceInfoPlugin
2018-12-27 17:56:15,373 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SolidView
2018-12-27 17:56:15,417 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SupportEraser
2018-12-27 17:56:15,507 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin Toolbox
2018-12-27 17:56:15,555 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin CameraTool
2018-12-27 17:56:15,604 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin MirrorTool
2018-12-27 17:56:15,658 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin RotateTool
2018-12-27 17:56:15,718 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin ScaleTool
2018-12-27 17:56:15,757 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SelectionTool
2018-12-27 17:56:15,812 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin TranslateTool
2018-12-27 17:56:15,888 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UFPWriter
2018-12-27 17:56:15,942 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UltimakerMachineActions
2018-12-27 17:56:16,202 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UM3NetworkPrinting
2018-12-27 17:56:16,258 - INFO - [Thread-1] UpdateChecker.UpdateCheckerJob.run [36]: Checking for new version of cura
2018-12-27 17:56:16,267 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UpdateChecker
2018-12-27 17:56:16,389 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin USBPrinting
2018-12-27 17:56:16,430 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UserAgreement
2018-12-27 17:56:16,494 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade21to22
2018-12-27 17:56:16,538 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade22to24
2018-12-27 17:56:16,587 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade25to26
2018-12-27 17:56:16,631 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade26to27
2018-12-27 17:56:16,683 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade27to30
2018-12-27 17:56:16,731 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade30to31
2018-12-27 17:56:16,777 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade32to33
2018-12-27 17:56:16,821 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade33to34
2018-12-27 17:56:16,867 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade34to35
2018-12-27 17:56:16,915 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SimpleView
2018-12-27 17:56:16,989 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin X3DReader
2018-12-27 17:56:17,073 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin XmlMaterialProfile
2018-12-27 17:56:17,127 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin XRayView
2018-12-27 17:56:17,172 - INFO - [MainThread] UM.VersionUpgradeManager.upgrade [130]: Looking for old configuration files to upgrade.
2018-12-27 17:56:17,185 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,193 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,203 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,211 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,220 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,228 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,238 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,246 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,256 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,264 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,276 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,285 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,295 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,303 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,316 - INFO - [MainThread] UM.Qt.QtApplication.startSplashWindowPhase [198]: The preferences file cannot be found, will use default values
2018-12-27 17:56:17,327 - DEBUG - [MainThread] UM.Backend.Backend._onSocketError [204]: Socket was unable to bind to port, increasing port number to 49675
2018-12-27 17:56:17,333 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:17,340 - DEBUG - [MainThread] CuraEngineBackend.CuraEngineBackend._terminate [293]: Attempting to kill the engine process
2018-12-27 17:56:17,347 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:17,362 - INFO - [MainThread] UM.Qt.QtApplication.startSplashWindowPhase [207]: The preferences file 'None' cannot be found, will use default values
2018-12-27 17:56:17,777 - DEBUG - [MainThread] UM.Backend.Backend._onSocketError [204]: Socket was unable to bind to port, increasing port number to 49676
2018-12-27 17:56:17,784 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:17,790 - DEBUG - [MainThread] CuraEngineBackend.CuraEngineBackend._terminate [293]: Attempting to kill the engine process
2018-12-27 17:56:17,797 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:18,349 - DEBUG - [MainThread] UM.Backend.Backend._onSocketError [204]: Socket was unable to bind to port, increasing port number to 49677
2018-12-27 17:56:18,356 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:18,361 - DEBUG - [MainThread] CuraEngineBackend.CuraEngineBackend._terminate [293]: Attempting to kill the engine process
2018-12-27 17:56:18,369 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:18,388 - DEBUG - [MainThread] UM.Backend.Backend._logSocketState [178]: Socket state changed to Listening
2018-12-27 17:56:18,404 - INFO - [MainThread] UM.Backend.Backend.startEngine [74]: Started engine process: C:\Program Files\Ultimaker Cura 3.6\CuraEngine.exe
2018-12-27 17:56:18,413 - DEBUG - [MainThread] UM.Backend.Backend._backendLog [94]: [Backend] Calling engine with: ['C:\\Program Files\\Ultimaker Cura 3.6\\CuraEngine.exe', 'connect', '127.0.0.1:49677', '']
2018-12-27 17:56:18,583 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] 
2018-12-27 17:56:18,598 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] Cura_SteamEngine version 3.6.0
2018-12-27 17:56:18,605 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] Copyright (C) 2018 Ultimaker
2018-12-27 17:56:18,615 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] 
2018-12-27 17:56:18,619 - DEBUG - [MainThread] UM.Backend.Backend._logSocketState [182]: Socket state changed to Connected
2018-12-27 17:56:18,626 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] This program is free software: you can redistribute it and/or modify
2018-12-27 17:56:18,633 - DEBUG - [MainThread] UM.Backend.Backend._onSocketStateChanged [172]: Backend connected on port 49677
2018-12-27 17:56:18,639 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] it under the terms of the GNU Affero General Public License as published by
2018-12-27 17:56:18,649 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] the Free Software Foundation, either version 3 of the License, or
2018-12-27 17:56:18,656 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] (at your option) any later version.
2018-12-27 17:56:18,663 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] 
2018-12-27 17:56:18,670 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] This program is distributed in the hope that it will be useful,
2018-12-27 17:56:18,677 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] but WITHOUT ANY WARRANTY; without even the implied warranty of
2018-12-27 17:56:18,684 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
2018-12-27 17:56:18,691 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] GNU Affero General Public License for more details.
2018-12-27 17:56:18,697 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] 
2018-12-27 17:56:18,704 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] You should have received a copy of the GNU Affero General Public License
2018-12-27 17:56:18,716 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] along with this program.  If not, see <http://www.gnu.org/licenses/>.
2018-12-27 17:56:22,827 - DEBUG - [MainThread] cura.Scene.CuraSceneController.setActiveBuildPlate [103]: Select build plate: 0
2018-12-27 17:56:22,835 - INFO - [MainThread] cura.CuraApplication.run [689]: Initializing variant manager
2018-12-27 17:56:22,854 - DEBUG - [MainThread] cura.Machines.VariantManager.initialize [59]: Exclude variant [empty_variant]
2018-12-27 17:56:22,857 - DEBUG - [MainThread] LocalContainerProvider.LocalContainerProvider.loadContainer [54]: Loading container ultimaker_s5_glass
2018-12-27 17:56:22,869 - DEBUG - [MainThread] UM.Settings.ContainerRegistry.addContainer [371]: Container [ultimaker_s5_glass] added.
2018-12-27 17:56:22,872 - DEBUG - [MainThread] LocalContainerProvider.LocalContainerProvider.loadContainer [54]: Loading container ultimaker_s5
2018-12-27 17:56:23,977 - DEBUG - [MainThread] UM.Settings.ContainerRegistry.addContainer [371]: Container [ultimaker_s5] added.
2018-12-27 17:56:23,986 - DEBUG - [MainThread] LocalContainerProvider.LocalContainerProvider.loadContainer [54]: Loading container ultimaker_s5_aluminum
2018-12-27 17:56:24,000 - DEBUG - [MainThread] UM.Settings.ContainerRegistry.addContainer [371]: Container [ultimaker_s5_aluminum] added.
2018-12-27 17:56:24,013 - INFO - [MainThread] cura.CuraApplication.run [693]: Initializing material manager
2018-12-27 17:56:24,279 - INFO - [MainThread] cura.CuraApplication.run [698]: Initializing quality manager
2018-12-27 17:56:24,323 - DEBUG - [MainThread] cura.Machines.QualityManager.initialize [136]: Lookup tables updated.
2018-12-27 17:56:24,330 - INFO - [MainThread] cura.CuraApplication.run [703]: Initializing machine manager
2018-12-27 17:56:24,341 - INFO - [MainThread] cura.CuraApplication.run [706]: Initializing container manager
2018-12-27 17:56:24,348 - INFO - [MainThread] cura.CuraApplication.run [709]: Initializing machine error checker
2018-12-27 17:56:25,891 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin._createView [279]: Creating post processing plugin view.
2018-12-27 17:56:25,905 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: BQ_PauseAtHeight
2018-12-27 17:56:25,914 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: ChangeAtZ
2018-12-27 17:56:25,919 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: DisplayFilenameAndLayerOnLCD
2018-12-27 17:56:25,922 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: ExampleScript
2018-12-27 17:56:25,927 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: FilamentChange
2018-12-27 17:56:25,934 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: PauseAtHeight
2018-12-27 17:56:25,938 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: PauseAtHeightRepRapFirmwareDuet
2018-12-27 17:56:25,944 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: PauseAtHeightforRepetier
2018-12-27 17:56:25,947 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: SearchAndReplace
2018-12-27 17:56:25,961 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: Stretch
2018-12-27 17:56:26,234 - DEBUG - [MainThread] UM.Qt.Bindings.Theme.load [172]: Loading theme file: C:\Program Files\Ultimaker Cura 3.6\resources\themes\cura-light\theme.json
2018-12-27 17:56:26,473 - DEBUG - [MainThread] UM.Qt.Bindings.Theme.load [244]: Loaded theme C:\Program Files\Ultimaker Cura 3.6\resources\themes\cura-light
2018-12-27 17:56:26,551 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin._createView [289]: Post processing view created.
2018-12-27 17:56:26,961 - DEBUG - [MainThread] LocalContainerProvider.LocalContainerProvider.loadContainer [54]: Loading container fdmprinter
2018-12-27 17:56:27,776 - DEBUG - [MainThread] UM.Settings.ContainerRegistry.addContainer [371]: Container [fdmprinter] added.
2018-12-27 17:56:27,823 - DEBUG - [MainThread] cura.Machines.Models.CustomQualityProfilesDropDownMenuModel._update [15]: Updating CustomQualityProfilesDropDownMenuModel.
2018-12-27 17:56:27,829 - DEBUG - [MainThread] cura.Machines.Models.CustomQualityProfilesDropDownMenuModel._update [20]: No active GlobalStack, set CustomQualityProfilesDropDownMenuModel as empty.
2018-12-27 17:56:27,837 - DEBUG - [MainThread] cura.Machines.Models.QualityProfilesDropDownMenuModel._update [50]: Updating QualityProfilesDropDownMenuModel.
2018-12-27 17:56:27,843 - DEBUG - [MainThread] cura.Machines.Models.QualityProfilesDropDownMenuModel._update [55]: No active GlobalStack, set quality profile model as empty.
2018-12-27 17:56:27,851 - DEBUG - [MainThread] cura.Machines.Models.BuildPlateModel._update [33]: Updating BuildPlateModel.
2018-12-27 17:56:29,602 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Topbar.qml:19: TypeError: Cannot read property 'stageId' of null
2018-12-27 17:56:29,609 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Topbar.qml:104: TypeError: Cannot read property 'stageId' of null
2018-12-27 17:56:29,616 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Topbar.qml:175: TypeError: Cannot read property 'stageId' of null
2018-12-27 17:56:29,622 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Cura.qml:444: TypeError: Cannot read property 'mainComponent' of null
2018-12-27 17:56:29,629 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Cura.qml:438: TypeError: Cannot read property 'mainComponent' of null
2018-12-27 17:56:29,636 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Cura.qml:476: TypeError: Cannot read property 'sidebarComponent' of null
2018-12-27 17:56:29,643 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Cura.qml:508: TypeError: Cannot read property 'sidebarComponent' of null
2018-12-27 17:56:29,671 - DEBUG - [MainThread] UM3NetworkPrinting.src.DiscoverUM3Action._createAdditionalComponentsView [171]: Creating additional ui components for UM3.
2018-12-27 17:56:29,793 - DEBUG - [MainThread] UM3NetworkPrinting.src.DiscoverUM3Action.startDiscovery [43]: Starting device discovery.
2018-12-27 17:56:29,906 - DEBUG - [MainThread] UM.Controller.setActiveStage [159]: Setting active stage to PrepareStage
2018-12-27 17:56:30,417 - DEBUG - [MainThread] cura.Machines.Models.NozzleModel._update [34]: Updating NozzleModel.
2018-12-27 17:56:30,685 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/SidebarHeader.qml:551:9: QML QQuickItem: Binding loop detected for property "width"
2018-12-27 17:56:30,768 - DEBUG - [MainThread] UM.Controller.setActiveView [92]: Setting active view to SolidView
2018-12-27 17:56:30,780 - DEBUG - [MainThread] ModelChecker.ModelChecker._createView [97]: Creating model checker view.
2018-12-27 17:56:30,801 - DEBUG - [MainThread] ModelChecker.ModelChecker._createView [106]: Model checker view created.
2018-12-27 17:56:30,806 - DEBUG - [MainThread] SliceInfoPlugin.SliceInfo._createDialog [82]: Creating dialog [MoreInfoWindow.qml]
2018-12-27 17:56:30,973 - DEBUG - [MainThread] cura.CuraApplication.run [741]: Booting Cura took 18.392673015594482 seconds
2018-12-27 17:56:31,554 - DEBUG - [MainThread] UM.Qt.QtRenderer._initialize [183]: Support for Vertex Array Objects: True
2018-12-27 17:56:31,579 - DEBUG - [MainThread] UM.View.GL.OpenGL.__init__ [99]: Initialized OpenGL subsystems.
2018-12-27 17:56:31,585 - DEBUG - [MainThread] UM.View.GL.OpenGL.__init__ [100]: OpenGL Version:  4.1.0 - Build 10.18.10.3277
2018-12-27 17:56:31,592 - DEBUG - [MainThread] UM.View.GL.OpenGL.__init__ [101]: OpenGL Vendor:   Intel
2018-12-27 17:56:31,599 - DEBUG - [MainThread] UM.View.GL.OpenGL.__init__ [102]: OpenGL Renderer: Intel(R) HD Graphics 4600
2018-12-27 17:56:31,608 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\default.shader]...
2018-12-27 17:56:31,633 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\selection.shader]...
2018-12-27 17:56:31,646 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\default.shader]...
2018-12-27 17:56:31,662 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\composite.shader]...
2018-12-27 17:56:31,696 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\overhang.shader]...
2018-12-27 17:56:31,720 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\striped.shader]...
2018-12-27 17:56:31,749 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\transparent_object.shader]...
2018-12-27 17:56:31,768 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\striped.shader]...
2018-12-27 17:56:31,799 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\platform.shader]...

**stderr.log -->**
2018-12-27 17:56:14,038 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin ConsoleLogger
2018-12-27 17:56:14,132 - INFO - [MainThread] CuraEngineBackend.CuraEngineBackend.__init__ [80]: Found CuraEngine at: C:\Program Files\Ultimaker Cura 3.6\CuraEngine.exe
2018-12-27 17:56:14,142 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin CuraEngineBackend
2018-12-27 17:56:14,175 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin CuraProfileReader
2018-12-27 17:56:14,207 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin CuraProfileWriter
2018-12-27 17:56:14,241 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin OBJReader
2018-12-27 17:56:14,277 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin OBJWriter
2018-12-27 17:56:14,328 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin STLReader
2018-12-27 17:56:14,365 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin STLWriter
2018-12-27 17:56:14,395 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin FileLogger
2018-12-27 17:56:14,448 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin FirmwareUpdateChecker
2018-12-27 17:56:14,485 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin FirmwareUpdater
2018-12-27 17:56:14,522 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeGzReader
2018-12-27 17:56:14,557 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeGzWriter
2018-12-27 17:56:14,593 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeProfileReader
2018-12-27 17:56:14,675 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeReader
2018-12-27 17:56:14,716 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin GCodeWriter
2018-12-27 17:56:14,765 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin ImageReader
2018-12-27 17:56:14,802 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin LegacyProfileReader
2018-12-27 17:56:14,840 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin LocalContainerProvider
2018-12-27 17:56:14,885 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin LocalFileOutputDevice
2018-12-27 17:56:14,920 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin MachineSettingsAction
2018-12-27 17:56:14,956 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin ModelChecker
2018-12-27 17:56:14,995 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin MonitorStage
2018-12-27 17:56:15,036 - WARNING - [MainThread] UM.Preferences.getValue [97]: Tried to get the value of non-existing setting cura/active_mode.
2018-12-27 17:56:15,046 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin PerObjectSettingsTool
2018-12-27 17:56:15,091 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin PostProcessingPlugin
2018-12-27 17:56:15,128 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin PrepareStage
2018-12-27 17:56:15,192 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin RemovableDriveOutputDevice
2018-12-27 17:56:15,277 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SimulationView
2018-12-27 17:56:15,326 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SliceInfoPlugin
2018-12-27 17:56:15,372 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SolidView
2018-12-27 17:56:15,417 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SupportEraser
2018-12-27 17:56:15,506 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin Toolbox
2018-12-27 17:56:15,555 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin CameraTool
2018-12-27 17:56:15,603 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin MirrorTool
2018-12-27 17:56:15,658 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin RotateTool
2018-12-27 17:56:15,718 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin ScaleTool
2018-12-27 17:56:15,756 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SelectionTool
2018-12-27 17:56:15,811 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin TranslateTool
2018-12-27 17:56:15,887 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UFPWriter
2018-12-27 17:56:15,941 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UltimakerMachineActions
2018-12-27 17:56:16,202 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UM3NetworkPrinting
2018-12-27 17:56:16,258 - INFO - [Thread-1] UpdateChecker.UpdateCheckerJob.run [36]: Checking for new version of cura
2018-12-27 17:56:16,267 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UpdateChecker
2018-12-27 17:56:16,388 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin USBPrinting
2018-12-27 17:56:16,429 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin UserAgreement
2018-12-27 17:56:16,494 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade21to22
2018-12-27 17:56:16,537 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade22to24
2018-12-27 17:56:16,587 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade25to26
2018-12-27 17:56:16,631 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade26to27
2018-12-27 17:56:16,683 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade27to30
2018-12-27 17:56:16,731 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade30to31
2018-12-27 17:56:16,777 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade32to33
2018-12-27 17:56:16,820 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade33to34
2018-12-27 17:56:16,867 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin VersionUpgrade34to35
2018-12-27 17:56:16,915 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin SimpleView
2018-12-27 17:56:16,989 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin X3DReader
2018-12-27 17:56:17,073 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin XmlMaterialProfile
2018-12-27 17:56:17,127 - INFO - [MainThread] UM.PluginRegistry.loadPlugin [422]: Loaded plugin XRayView
2018-12-27 17:56:17,172 - INFO - [MainThread] UM.VersionUpgradeManager.upgrade [130]: Looking for old configuration files to upgrade.
2018-12-27 17:56:17,185 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,193 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,202 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,210 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,220 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,228 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,237 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,246 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,256 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,264 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,275 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,285 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,295 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\packages.json, skip it
2018-12-27 17:56:17,303 - WARNING - [MainThread] UM.VersionUpgradeManager._getUpgradeTasks [281]: Failed to get file version: .\plugins.json, skip it
2018-12-27 17:56:17,315 - INFO - [MainThread] UM.Qt.QtApplication.startSplashWindowPhase [198]: The preferences file cannot be found, will use default values
2018-12-27 17:56:17,326 - DEBUG - [MainThread] UM.Backend.Backend._onSocketError [204]: Socket was unable to bind to port, increasing port number to 49675
2018-12-27 17:56:17,333 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:17,340 - DEBUG - [MainThread] CuraEngineBackend.CuraEngineBackend._terminate [293]: Attempting to kill the engine process
2018-12-27 17:56:17,347 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:17,361 - INFO - [MainThread] UM.Qt.QtApplication.startSplashWindowPhase [207]: The preferences file 'None' cannot be found, will use default values
2018-12-27 17:56:17,776 - DEBUG - [MainThread] UM.Backend.Backend._onSocketError [204]: Socket was unable to bind to port, increasing port number to 49676
2018-12-27 17:56:17,783 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:17,789 - DEBUG - [MainThread] CuraEngineBackend.CuraEngineBackend._terminate [293]: Attempting to kill the engine process
2018-12-27 17:56:17,796 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:18,348 - DEBUG - [MainThread] UM.Backend.Backend._onSocketError [204]: Socket was unable to bind to port, increasing port number to 49677
2018-12-27 17:56:18,355 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:18,361 - DEBUG - [MainThread] CuraEngineBackend.CuraEngineBackend._terminate [293]: Attempting to kill the engine process
2018-12-27 17:56:18,369 - DEBUG - [MainThread] UM.Backend.Backend._createSocket [218]: Previous socket existed. Closing that first.
2018-12-27 17:56:18,388 - DEBUG - [MainThread] UM.Backend.Backend._logSocketState [178]: Socket state changed to Listening
2018-12-27 17:56:18,404 - INFO - [MainThread] UM.Backend.Backend.startEngine [74]: Started engine process: C:\Program Files\Ultimaker Cura 3.6\CuraEngine.exe
2018-12-27 17:56:18,413 - DEBUG - [MainThread] UM.Backend.Backend._backendLog [94]: [Backend] Calling engine with: ['C:\\Program Files\\Ultimaker Cura 3.6\\CuraEngine.exe', 'connect', '127.0.0.1:49677', '']
2018-12-27 17:56:18,582 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] 
2018-12-27 17:56:18,594 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] Cura_SteamEngine version 3.6.0
2018-12-27 17:56:18,605 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] Copyright (C) 2018 Ultimaker
2018-12-27 17:56:18,615 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] 
2018-12-27 17:56:18,618 - DEBUG - [MainThread] UM.Backend.Backend._logSocketState [182]: Socket state changed to Connected
2018-12-27 17:56:18,625 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] This program is free software: you can redistribute it and/or modify
2018-12-27 17:56:18,632 - DEBUG - [MainThread] UM.Backend.Backend._onSocketStateChanged [172]: Backend connected on port 49677
2018-12-27 17:56:18,639 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] it under the terms of the GNU Affero General Public License as published by
2018-12-27 17:56:18,649 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] the Free Software Foundation, either version 3 of the License, or
2018-12-27 17:56:18,656 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] (at your option) any later version.
2018-12-27 17:56:18,663 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] 
2018-12-27 17:56:18,670 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] This program is distributed in the hope that it will be useful,
2018-12-27 17:56:18,677 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] but WITHOUT ANY WARRANTY; without even the implied warranty of
2018-12-27 17:56:18,684 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
2018-12-27 17:56:18,690 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] GNU Affero General Public License for more details.
2018-12-27 17:56:18,697 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] 
2018-12-27 17:56:18,704 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] You should have received a copy of the GNU Affero General Public License
2018-12-27 17:56:18,716 - DEBUG - [Thread-20] UM.Backend.Backend._backendLog [94]: [Backend] along with this program.  If not, see <http://www.gnu.org/licenses/>.
2018-12-27 17:56:22,827 - DEBUG - [MainThread] cura.Scene.CuraSceneController.setActiveBuildPlate [103]: Select build plate: 0
2018-12-27 17:56:22,834 - INFO - [MainThread] cura.CuraApplication.run [689]: Initializing variant manager
2018-12-27 17:56:22,853 - DEBUG - [MainThread] cura.Machines.VariantManager.initialize [59]: Exclude variant [empty_variant]
2018-12-27 17:56:22,857 - DEBUG - [MainThread] LocalContainerProvider.LocalContainerProvider.loadContainer [54]: Loading container ultimaker_s5_glass
2018-12-27 17:56:22,869 - DEBUG - [MainThread] UM.Settings.ContainerRegistry.addContainer [371]: Container [ultimaker_s5_glass] added.
2018-12-27 17:56:22,872 - DEBUG - [MainThread] LocalContainerProvider.LocalContainerProvider.loadContainer [54]: Loading container ultimaker_s5
2018-12-27 17:56:23,977 - DEBUG - [MainThread] UM.Settings.ContainerRegistry.addContainer [371]: Container [ultimaker_s5] added.
2018-12-27 17:56:23,986 - DEBUG - [MainThread] LocalContainerProvider.LocalContainerProvider.loadContainer [54]: Loading container ultimaker_s5_aluminum
2018-12-27 17:56:24,000 - DEBUG - [MainThread] UM.Settings.ContainerRegistry.addContainer [371]: Container [ultimaker_s5_aluminum] added.
2018-12-27 17:56:24,013 - INFO - [MainThread] cura.CuraApplication.run [693]: Initializing material manager
2018-12-27 17:56:24,279 - INFO - [MainThread] cura.CuraApplication.run [698]: Initializing quality manager
2018-12-27 17:56:24,323 - DEBUG - [MainThread] cura.Machines.QualityManager.initialize [136]: Lookup tables updated.
2018-12-27 17:56:24,329 - INFO - [MainThread] cura.CuraApplication.run [703]: Initializing machine manager
2018-12-27 17:56:24,340 - INFO - [MainThread] cura.CuraApplication.run [706]: Initializing container manager
2018-12-27 17:56:24,347 - INFO - [MainThread] cura.CuraApplication.run [709]: Initializing machine error checker
2018-12-27 17:56:25,891 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin._createView [279]: Creating post processing plugin view.
2018-12-27 17:56:25,905 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: BQ_PauseAtHeight
2018-12-27 17:56:25,914 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: ChangeAtZ
2018-12-27 17:56:25,919 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: DisplayFilenameAndLayerOnLCD
2018-12-27 17:56:25,922 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: ExampleScript
2018-12-27 17:56:25,927 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: FilamentChange
2018-12-27 17:56:25,934 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: PauseAtHeight
2018-12-27 17:56:25,938 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: PauseAtHeightRepRapFirmwareDuet
2018-12-27 17:56:25,943 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: PauseAtHeightforRepetier
2018-12-27 17:56:25,947 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: SearchAndReplace
2018-12-27 17:56:25,961 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin.loadScripts [169]: Begin loading of script: Stretch
2018-12-27 17:56:26,233 - DEBUG - [MainThread] UM.Qt.Bindings.Theme.load [172]: Loading theme file: C:\Program Files\Ultimaker Cura 3.6\resources\themes\cura-light\theme.json
2018-12-27 17:56:26,473 - DEBUG - [MainThread] UM.Qt.Bindings.Theme.load [244]: Loaded theme C:\Program Files\Ultimaker Cura 3.6\resources\themes\cura-light
2018-12-27 17:56:26,550 - DEBUG - [MainThread] PostProcessingPlugin.PostProcessingPlugin._createView [289]: Post processing view created.
2018-12-27 17:56:26,961 - DEBUG - [MainThread] LocalContainerProvider.LocalContainerProvider.loadContainer [54]: Loading container fdmprinter
2018-12-27 17:56:27,776 - DEBUG - [MainThread] UM.Settings.ContainerRegistry.addContainer [371]: Container [fdmprinter] added.
2018-12-27 17:56:27,822 - DEBUG - [MainThread] cura.Machines.Models.CustomQualityProfilesDropDownMenuModel._update [15]: Updating CustomQualityProfilesDropDownMenuModel.
2018-12-27 17:56:27,829 - DEBUG - [MainThread] cura.Machines.Models.CustomQualityProfilesDropDownMenuModel._update [20]: No active GlobalStack, set CustomQualityProfilesDropDownMenuModel as empty.
2018-12-27 17:56:27,836 - DEBUG - [MainThread] cura.Machines.Models.QualityProfilesDropDownMenuModel._update [50]: Updating QualityProfilesDropDownMenuModel.
2018-12-27 17:56:27,843 - DEBUG - [MainThread] cura.Machines.Models.QualityProfilesDropDownMenuModel._update [55]: No active GlobalStack, set quality profile model as empty.
2018-12-27 17:56:27,851 - DEBUG - [MainThread] cura.Machines.Models.BuildPlateModel._update [33]: Updating BuildPlateModel.
2018-12-27 17:56:29,602 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Topbar.qml:19: TypeError: Cannot read property 'stageId' of null
2018-12-27 17:56:29,609 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Topbar.qml:104: TypeError: Cannot read property 'stageId' of null
2018-12-27 17:56:29,615 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Topbar.qml:175: TypeError: Cannot read property 'stageId' of null
2018-12-27 17:56:29,622 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Cura.qml:444: TypeError: Cannot read property 'mainComponent' of null
2018-12-27 17:56:29,629 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Cura.qml:438: TypeError: Cannot read property 'mainComponent' of null
2018-12-27 17:56:29,636 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Cura.qml:476: TypeError: Cannot read property 'sidebarComponent' of null
2018-12-27 17:56:29,643 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/Cura.qml:508: TypeError: Cannot read property 'sidebarComponent' of null
2018-12-27 17:56:29,670 - DEBUG - [MainThread] UM3NetworkPrinting.src.DiscoverUM3Action._createAdditionalComponentsView [171]: Creating additional ui components for UM3.
2018-12-27 17:56:29,793 - DEBUG - [MainThread] UM3NetworkPrinting.src.DiscoverUM3Action.startDiscovery [43]: Starting device discovery.
2018-12-27 17:56:29,906 - DEBUG - [MainThread] UM.Controller.setActiveStage [159]: Setting active stage to PrepareStage
2018-12-27 17:56:30,417 - DEBUG - [MainThread] cura.Machines.Models.NozzleModel._update [34]: Updating NozzleModel.
2018-12-27 17:56:30,685 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [352]: file:///C:/Program Files/Ultimaker Cura 3.6/resources/qml/SidebarHeader.qml:551:9: QML QQuickItem: Binding loop detected for property "width"
2018-12-27 17:56:30,767 - DEBUG - [MainThread] UM.Controller.setActiveView [92]: Setting active view to SolidView
2018-12-27 17:56:30,779 - DEBUG - [MainThread] ModelChecker.ModelChecker._createView [97]: Creating model checker view.
2018-12-27 17:56:30,801 - DEBUG - [MainThread] ModelChecker.ModelChecker._createView [106]: Model checker view created.
2018-12-27 17:56:30,806 - DEBUG - [MainThread] SliceInfoPlugin.SliceInfo._createDialog [82]: Creating dialog [MoreInfoWindow.qml]
2018-12-27 17:56:30,972 - DEBUG - [MainThread] cura.CuraApplication.run [741]: Booting Cura took 18.392673015594482 seconds
2018-12-27 17:56:31,554 - DEBUG - [MainThread] UM.Qt.QtRenderer._initialize [183]: Support for Vertex Array Objects: True
2018-12-27 17:56:31,578 - DEBUG - [MainThread] UM.View.GL.OpenGL.__init__ [99]: Initialized OpenGL subsystems.
2018-12-27 17:56:31,585 - DEBUG - [MainThread] UM.View.GL.OpenGL.__init__ [100]: OpenGL Version:  4.1.0 - Build 10.18.10.3277
2018-12-27 17:56:31,592 - DEBUG - [MainThread] UM.View.GL.OpenGL.__init__ [101]: OpenGL Vendor:   Intel
2018-12-27 17:56:31,598 - DEBUG - [MainThread] UM.View.GL.OpenGL.__init__ [102]: OpenGL Renderer: Intel(R) HD Graphics 4600
2018-12-27 17:56:31,607 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\default.shader]...
2018-12-27 17:56:31,632 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\selection.shader]...
2018-12-27 17:56:31,645 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\default.shader]...
2018-12-27 17:56:31,662 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\composite.shader]...
2018-12-27 17:56:31,695 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\overhang.shader]...
2018-12-27 17:56:31,719 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\striped.shader]...
2018-12-27 17:56:31,749 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\transparent_object.shader]...
2018-12-27 17:56:31,768 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\striped.shader]...
2018-12-27 17:56:31,798 - DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Program Files\Ultimaker Cura 3.6\resources\shaders\platform.shader]...

stdout.log -->
Empty

**Comment by Ellecross** [comment]
Hey @efyoung 

Thanks for the info! Good information provided as well on the platform you're using. I was going to ask those at first. :D 

So, a few questions: 

1. Did you happen to have an older version of Cura installed? What version were you using? 
2. Can you zip and send in your configuration folder please? You can find it in \AppData\Roaming\Cura -> Please zip your whole Cura folder. 

This way, we can try and reproduce your scenario. ;)

**Comment by efyoung** [comment]
1. I installed 3.2 first from the website in which I bought the printer (https://downloads.monoprice.com/files/software/15365_Software_160615.zip), which seemed to work. I made it past the splash screen and agreement, then it asked me to pick my 3D printer, which I did (I chose custom), then before it went to the actual program, it said that there was an update and if I wanted to install. I did. The version that it installed was 3.4. That version did not work, and it gave the error as it did above, so I searched to see if there was a more recent version, and saw that version 3.6.0 was available, so I installed it. However, I still had the issue of it not starting. So, I uninstalled all versions and deleted all instances in the /%user%/AppData/ and reinstalled version 3.6.0. Still I could not get past the splash/user agreement.

2. 
[cura.zip](https://github.com/Ultimaker/Cura/files/2715519/cura.zip)

**Comment by Ellecross** [comment]
Thank you for the additional information! 

About that link you posted, the Cura version in it was really old 15.04.2 - and not 3.2. But, I did try the scenario you are talking about on an old machine and with my own and I still cannot reproduce it. 

1. Installed 3.2
2. Added Custom printer
3. Updated to 3.4 
4. Closed 3.2 and opened 3.4 -> All OK 
5. Updated and opened 3.6 -> All OK 
6. I tried with 4.0 Beta as well and it was OK. :( 

Anyone else may have an idea on what's going on? 

I'll put in the Cura discussion if any one else in our team has an idea. Sorry for now. :/

**Comment by youngef** [comment]
Strange. I just wiped all versions, including the AppData folder and installed 3.6 I thought. I’ll wipe everything again and reinstall and repost my findings today.

**Comment by efyoung** [comment]
So, I uninstalled version 3.6 that I confirmed that I installed last time. I deleted the AppData cura folder. I rebooted and installed Ultimaker_Cura-3.6.0-win64.exe. Still have the same problem. Here is the cura folder from the AppData roaming folder:
[cura.zip](https://github.com/Ultimaker/Cura/files/2718838/cura.zip)

**Comment by Ellecross** [comment]
hey @efyoung ! 

I got your roaming folder, thanks! I cannot reproduce it either. :/ 3.6 and 4.0 starts properly for me. There are no printers yet in that configuration folder, so Cura shows up the Add Printer dialogue as expected. 

Sorry I'm gonna have to ask you to try a few more things: 

1. Can you please try with Cura 4.0 Beta, just to see if it's the same issue? 
2. I do not see any errors in the cura stderr.log file, but do you see any weird error messages in Windows Event Viewer itself? 

Thanks and let me know what happens!

**Comment by efyoung** [comment]
So, I uninstalled SEP, rebooted, and ran Cura 4.0. Now I am getting a Windows Firewall popup that says Windows Firewall has blocked some features of cura.exe on all public and private networks. It asks me to allow cura.exe to communicate on these networks, giving me the option to select private networks, such as my home or work network, and public networks, such as those in airports and coffee shops (not recommended because these networks often have little or no security). While waiting to make my selection, Cura crashes. So does Cura need network access private or public? I will be installing a different antivirus (Mcafee) program. I'm going to select private networks and see if that allows Cura to open and not crash.

Update: I selected private network, opened Cura, and it still crashed. The Windows Event Viewer has the same Application Error, which is below:

Faulting application name: Cura.exe, version: 0.0.0.0, time stamp: 0x5c08ef53
Faulting module name: ig75icd64.dll, version: 10.18.10.3277, time stamp: 0x5212a803
Exception code: 0xc0000005
Fault offset: 0x000000000026d0c2
Faulting process id: 0x118c
Faulting application start time: 0x01d4a2f3b01fb4cb
Faulting application path: C:\Program Files\Ultimaker Cura 4.0\Cura.exe
Faulting module path: C:\Windows\SYSTEM32\ig75icd64.dll
Report Id: fa9ad96f-0ee6-11e9-8301-a0d3c1c91697
Faulting package full name: 
Faulting package-relative application ID:

**Comment by efyoung** [comment]
Problem averted. I updated Windows to 10 and now it works. I still have no idea what was wrong with the Windows 8.1 not working. Glad to have Cura working now. Thank you for all your assistance.

**Comment by nallath** [comment]
Cura uses a local socket to connect with the backend. That's the only connection it really needs to actually do what it needs to do. There is also the update checker (which just checks what the latest version is and compares it with it's own version), but that's not actually required to work.

For some reason, despite us trying to get Cura whitelisted, every so often a number of firewalls go haywire and label Cura as a trojan-like. The people that make the virus software simply refuse to explain why this happens or what we can do to prevent this.

**Comment by Ellecross** [comment]
> 
> 
> Problem averted. I updated Windows to 10 and now it works. I still have no idea what was wrong with the Windows 8.1 not working. Glad to have Cura working now. Thank you for all your assistance.

Great! That's really great to know! Maybe with your update to Windows 10, other things have been updated as well.

**Comment by denmorru** [comment]
I have latest version of cura 4.0.0 released 
and Windows 10 x64 
- and it crashes exactly the same way on startup as mentioned above with last log record:
DEBUG - [MainThread] UM.View.GL.ShaderProgram.load [55]: Loading shader file [C:\Cura\resources\shaders\platform.shader]...

**Comment by Ghostkeeper** [comment]
Better create a new ticket for that then, or find one that is open.

Crashing after loading the shader files generally indicates a bug in our front-end that executes upon displaying the main interface. After those shader files it's going to load the main interface which is huge and probably has lots of bugs, so it's very hard to tell anything from just the fact that it crashes there.

**Comment by denmorru** [comment]
I found that the reason was something about how Cura (not)works with graphic adapter driver. 
In my case computer has two different video adapters managed by BMC device. Solution was: open second adapter (Nvidia control panel) settings and force Cura.exe to take user-defined 3D settings and exactly one adapter.

**Comment by bigleague1** [comment]
I run cura 3.6.18 lulzbot edition on my taz 6 and have had this issue as well even after I searched my entire PC for "cura" and deleted everything then did a reinstall and still had this issue with a user agreement UI crash. I work with computers and electronics professionally so I do have a tiny bit more experience with trouble shooting issues. When I looked over my files under the programs files for cura I noticed something that could be the cause of my problems and when it fixed the issue for me (after a lot of delete and install sessions with cura) I felt I had to share with the community just how to fix it. Sorry to ramble on, so without further delay, here is how I fixed it and it's really simple and anyone can do this.



1) navigate to your program files and open your cura files up.

2)Scroll down towards the bottom of the files and look for a folder (it's a zipped folder with the zipper icon going down the center of the folder) that is called "python35".

3)Right click the zipped folder and select the option "Extract All..." with a left click while you hover your cursor over it.

You may need to make a new folder named "python35" in the programs files for cura if you get  a popup or error message, but if you don't that is fine so long as the extracted "python35" normal folder pops up in the files in cura program files. If you do get a issue from extracting just back out and create the folder mentioned above then retry to extract from the zipped folder just like the steps above.

4)Verify the new folder was created and that it is not a zipped folder (this is something the program needs to run as far as I can tell) so once that's done you should be good to go.

5)run the program as you would normally and verify everything is working.

6)If your program loaded the next step is to stand straight up and jump up and down a minimum of 2 times to show your excitement that it is up and running ONCE AGAIN! if it did not work you have 2 options.



1) start over and make sure to follow instructions as mentioned above with greater care or if that doesn't work..

2)First uninstall your "cura" program first and then open up your computer panel (the place that shows you all the drives and removable media) and in the search area (on windows 10 it is towards the top right) and be sure to delete all files that have "cura" in it that one aren't repetitive, since you may have several results that are the same file so delete them as they show up in the search to limit the search results) but DON'T DELETE files that have the word "cura" in it since these can contain files used for other programs you may need so not just the "cura" name alone found somewhere.

2a) Once the program in uninstalled and the search is done with NO remaining"cura" file results found you can continue back to the start.



NOTE: this may not work in every situation and or cura version or model IE "cura lulzbot edition etc. If that is the case yo have a larer issue/s at hand and may want to contact whoever the software was modified IE lulzbot, Ultimaker etc for further assistance.
